packages/block-library/src/table/theme.scss

```diff
@@ -7,4 +7,10 @@
 		border: 1px solid;
 		word-break: normal;
 	}
+
+	caption {
```

Comment:
This rule is obsolete, right?

Reply:
Ah, yes. The selector should be `figcaption`. I've changed it over and removed some of the styling that's not needed any more.